product_docs/docs/pgd/5/appusage/extensions.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Using other extensions with PGD
+navTitle: Extension usage
+---
+
+## PGD and other PostgreSQL extensions
+
+PGD is implemented as a PostgreSQL extension (see [Supported Postgres database servers
+](overview/#supported-postgres-database-servers)), taking advantage of PostgreSQL's
+expandability and flexibility to modify system behaviour at quite a low level.
+
+In principle other extensions - both those provided by community PostgreSQL and
+EPAS, as well as third-party extensions, can be used together with PGD, however
+the distributed nature of PGD means selection and installation of extensions
+should be carefully considered and planned.
+
+### Extensions providing logical decoding
+
+Extensions providing logical decoding, such as [wal2json](https://github.com/eulerto/wal2json),
+may in theory work with PGD, but there is no support for failover, meaning any WAL stream
+being read from such an extension may be interrupted.
+
+### Extensions providing replication and/or HA functionality
+
+Any extension extending PostgreSQL with functionality related to replication
+and/or HA/failover is unlikely to work well with PGD, and may even be detrimental
+to the health of the PGD cluster, so should be avoided.
+
+### Supported extensions
```

Comment:
I'd bump this up the headings to H2 (and contents H3) and add deepToC: true to the frontmatter to make this more accessible.